### ISSUE TITLE
the depth of transformer blocks in PointTransformer

### ISSUE THREAD

**Issue by whu-lyh** [lulutang0608/Point-BERT#64]
Hi @yuxumin, is there a pyramid design for block depth at [here](https://github.com/lulutang0608/Point-BERT/blob/c76a4185feaafa523a54a29802f2f98b22ad35f7/segmentation/models/PointTransformer.py#L164)?

**Comment by yuxumin** [comment]
No, we use a pure sa with no pyramid design in all of our models.
